extras/create_send_notifications.py

```diff
@@ -42,26 +42,38 @@ def create_output_directories():
         os.path.join(NOTIFICATIONS_BASE_DIR, NOTIFICATION_ARCHIVE_DIR)
     )
 
+def build_notifications_org_list(db):
+    """Return notifications organization list.
 
-def build_cyhy_org_list(db):
-    """Build list of CyHy organization IDs.
-
-    This is the list of CyHy organization IDs (and their descendants) that
-    receive CyHy reports.
+    This is the list of organization IDs that should
+    get a notification PDF for CyHy report types.
```

Comment:
No need to mention that the notification is a PDF or that the orgs have "CYHY" in their `report_types`:
```suggestion
    have a notification generated and sent.
```